app/actors/hyrax/actors/attach_members_actor.rb

```diff
@@ -30,10 +30,16 @@ def assign_nested_attributes_for_collection(env, attributes_collection)
           # checking for existing works to avoid rewriting/loading works that are
           # already attached
           existing_works = env.curation_concern.member_ids
+          little_boolean = ActiveModel::Type::Boolean.new
```

Comment:
Can you name this something like `boolean_type`?  The term little can mean small, but in this case, I assume you are using it as meaning lower_case.